sdk/identity/azure-identity/setup.py

```diff
@@ -60,7 +60,7 @@
     install_requires=[
         "azure-core<2.0.0,>=1.11.0",
         "cryptography>=2.5",
-        "msal<2.0.0,>=1.12.0",
+        "msal<2.0.0,>=1.20.0",
```

Comment:
There is a tricky thing here. [MSAL Python chose to declare the broker (i.e. WAM in this context) dependency as extra](https://github.com/AzureAD/microsoft-authentication-library-for-python/blob/1.20.0/setup.py#L89-L96).

Azure Identity may choose to also expose an [extra functionality](https://peps.python.org/pep-0508/#extras) as `azure-identity[broker]` and then leave the decision for app developers. Alternatively, Azure Identity can simply always pull in MSAL's broker bits by `msal[broker]>=1.20.0,<2`.